README.md

```diff
@@ -171,6 +171,27 @@ This performs the same initial heating, before running a further 1000 steps (1 p
 When MD is run with heating the trajectory ```NaCl-nvt-T20.0-T300.0-T300.0-traj.xyz``` and statistics ```NaCl-nvt-T20.0-T300.0-T300.0-stats.dat``` files will indicate the heating range and MD temperature (which may be different). With heating and MD trajectories/statistics within the same files.
 
 
+### Phonons
+
+Calculate phonons with a 2x3x4 supercell, after geometry optimization (using the [MACE-MP](https://github.com/ACEsuit/mace-mp):
+
+```shell
+janus phonons --struct tests/data/NaCl.cif --supercell 2 --supercell 3 --supercell 4 --minimize --arch mace_mp --calc-kwargs "{'model' : 'small'}"
+```
+
+This will save the Phonopy parameters, including displacements and force constants, to `NaCl-params.yml`, and the calculated band structure to `NaCl-auto-band.yml`, in addition to generating a log file, `phonons.log`, and summary of inputs, `phonons_summary.yml`.
+
+Further calculations, including thermal properties, DOS, and PDOS, can also be calculated (using a 2x2x2 supercell):
+
+```shell
+janus phonons --struct tests/data/NaCl.cif --supercell 2 --dos --pdos --thermal --temp-start 0 --temp-end 300 --temp-step 50
```

Comment:
This also makes it more explicit here, because you would have to write `2x2x2` which is not a lot of extra typing